webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/getFileValueGraphDB.scala.txt

```diff
@@ -38,40 +35,24 @@ PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
 
-SELECT ?fileValue ?objPred ?objObj
+CONSTRUCT {
```

Comment:
@benjamingeer Could you test that with GraphDB locally?

Reply:
This doesn't seem possible with the current Bazel setup.

Reply:
Ideally, there would be a config option.

Reply:
I think our policy at the moment is that we're not officially maintaining support for GraphDB. But I'm trying to do it anyway if it's not too much effort.

Reply:
maybe at some point we might have more than one free triplestore available